wordpress-mobile/WordPress-Android
# Sign out of all sites with Logout button on Me tab

Fixes #3313 by replacing the existing `Disconnect from WordPress.com` button on the Me tab with a `Logout` button signs out of *all* sites, self hosted and WP.com.

A `Connect to WordPress.com` row has been added to let users who are signed into a self hosted site access their WP.com site. This new button is hidden when users are logged in. Previously the `Disconnect` button was dual purpose, connecting and disconnecting.

To test:
* Login to a self hosted site and head to the Me tab
* Verify both `Logout` and `Connect to WordPress.com` rows are visible
* Login to a WordPress.com account (without logging out of self hosted)
* Verify that only `Logout` is visible
* Select `Logout` and verify no sites are saved

cc @oguzkocer
